homeassistant/components/roborock/strings.json

```diff
@@ -267,5 +267,22 @@
         }
       }
     }
+  },
+  "exceptions": {
+    "command_failed": {
+      "message": "Error while calling {command} with {params}."
+    },
+    "home_data_fail": {
+      "message": "Failed getting Roborock home_data."
```

Comment:
`home_data` is very technical, while these are meant to be user-facing messages.

Reply:
changed to home data